.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
+
+  # Maintain dependencies for Docker
+  - package-ecosystem: docker
```

Comment:
Why this if the repo does not contain dockerfiles

Reply:
fixed, Thanks